build/yamls/antrea-eks.yml

```diff
@@ -579,6 +579,18 @@ data:
 
     # Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener.
     #enablePrometheusMetrics: false
+
+    # Provide flow collector address as string with format IP:port:L4(tcp or udp). This also enables flow exporter that sends IPFIX
```

Comment:
consider using this for clarity:
 `<IP>:<port>[:<proto>]`, where <proto> is tcp or udp.

Reply:
done